front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -92,15 +93,33 @@ export default function AssistantNew({
   const [conversation, setConversation] = useState<ConversationType | null>(
     null
   );
-  const { agentConfigurations } = useAgentConfigurations({
+
+  const globalAgentConfigurations = useAgentConfigurations({
```

Comment:
@philipperolet, is there a historical reason why using `view=global` doesn't filter to show only active assistants?

Reply:
Seen IRL with Henry.